smart-contracts/contracts/PublicLock.sol

```diff
@@ -1,11 +1,9 @@
 pragma solidity 0.5.13;
 
-import './interfaces/IERC721.sol';
-import './interfaces/IERC721Enumerable.sol';
+
 import './interfaces/IPublicLock.sol';
 import '@openzeppelin/contracts-ethereum-package/contracts/ownership/Ownable.sol';
 import '@openzeppelin/contracts-ethereum-package/contracts/introspection/ERC165.sol';
-import '@openzeppelin/upgrades/contracts/Initializable.sol';
```

Comment:
Why remove initializable?

Reply:
just some cleanup... `PublicLock.sol` already inherits `Initializable.sol` via both `IPublicLock.sol` and `MixinERC721Enumerable.sol`